test/ArcxAnalyticsSdk.test.ts

```diff
@@ -467,7 +467,7 @@ describe('(unit) ArcxAnalyticsSdk', () => {
         expect(eventStub).to.not.have.been.called
       })
 
-      it('catches click event', () => {
+      xit('catches click event', () => {
```

Comment:
To be unskipped in the next pr? or if it's too difficult, the test "makes track clicks on element without defined classname and id" in `ArcxAnalyticsProvider.test.tsx` can be enough

Reply:
I rebased and made some changes :) So that's your turn to review :)